corehq/motech/repeaters/models.py

```diff
@@ -258,8 +295,12 @@ def reset_next_attempt(self):
             self.next_attempt_at = None
             self.save()
 
+    def get_repeater_type(self):
```

Comment:
I have been fiddling around with the idea but was not convinced that whether it is a good idea to have a property `repeater_type` where we have also have an attribute of the same name. 

Maybe we can have a protected property `_repeater_type` instead of `repeater_type` but given the usage of the function, I am leaning towards keeping it a function rather than a property.

I would like to know what you think about it.

Reply:
Oh, I hand not noticed that this object had both a `repeater_type` attribute as well as this function because the attribute is only created/assigned in the `save()` method. I think it will be confusing to have both since it is not obvious to someone new to the code whether they should use `get_repeater_type()` or `repeater_type`. My preference would be to have only one (calculated) property named `repeater_type`.

If `save()` does not persist the calculated `repeater_type` property to the database then my second choice would be to have a private `_repeater_type` calculated property replacing this method and a `repeater_type` attribute that is assigned in `save()` (or earlier if possible) and always be used in public contexts.

Reply:
This has been addressed [here](https://github.com/dimagi/commcare-hq/pull/30755/files#diff-dbb494984fac15fda3d29dabadf40c26c18935eaac6d96c308d52d1f7237eb8cR296)